src/dotnet-dev-certs/Program.cs

```diff
@@ -95,6 +107,22 @@ public static int Main(string[] args)
             }
         }
 
+        private static int CleanHttpsCertificates(CommandOption clean, IReporter reporter)
```

Comment:
Where is `clean` coming from? You don't pass it in?

Reply:
Removed